src/core/libraries/kernel/threads/pthread.h

```diff
@@ -227,7 +238,8 @@ enum class ThreadListFlags : u32 {
     InGcList = 4,
 };
 
-using PthreadEntryFunc = void* (*)(void*);
+using PthreadEntryFunc = void* PS4_SYSV_ABI (*)(void*);
+using PthreadEntryHostFunc = void* (*)(void*);
```

Comment:
Feel like the use cases for this should just be made `PS4_SYSV_ABI`, having the same function take a function arg that can be two different ABIs does not sound like a good idea.